platformio.ini

```diff
@@ -81,31 +81,33 @@ lib_ldf_mode = chain+
 lib_deps = 
   https://github.com/emsesp/EMS-ESP-Modules.git
 
+;
 ; build for GitHub Actions CI
-; the Web interface is built seperately, so is skipped in extra_scripts
-[env:ci]
-; 4MB using Tasmota (no SSL) - like BBQKees older S32
+;
+; the Web interface is built seperately during the GH Action so is skipped (not included in extra_scripts)
+;
+[env:ci_s_4M]
+; 4MB ESP32 - using Tasmota - no SSL, no PSRAM - like the BBQKees older S32 models
 extends = espressi32_base_tasmota
 extra_scripts = scripts/rename_fw.py
 board = esp32dev
 board_build.partitions = esp32_partition_4M.csv
-board_build.extra_flags = -DBOARD_HAS_PSRAM
+build_flags =
+  ${espressi32_base_tasmota.build_flags}
+  '-DEMSESP_DEFAULT_BOARD_PROFILE="S32"'
```

Comment:
ci_s_4M could be S32 or E32 gateway, there should not be a defined DEFAULT_BOARD_PROFILE to trgger the automatic detection of E32 ethernet.

Reply:
good spot. I'll change!